package.json

```diff
@@ -125,8 +129,10 @@
     "arweave": "1.14.4",
     "axios": "1.7.7",
     "axios-retry": "^4.3.0",
+    "commander": "^12.1.0",
```

Comment:
can these be dev types? how does cmomander/prompts create the built CLI? is it a bundled file?

Reply:
looks like they are runtime deps, disregard. we can look into minimizing the build tree another time

Reply:
so far, we have them marked as external in the `esbuild` web bundle. but that doesn't excise the dependencies when using the library from npm package

Reply:
I feel the way to do that is to have actually publish a separate cli package, which isn't what we're intending here

Reply:
fwiw we could soon remove the deps in favor of node native CLI in future node releases